lib/Doctrine/DBAL/Types/JsonType.php

```diff
@@ -43,6 +43,10 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
     public function convertToDatabaseValue($value, AbstractPlatform $platform)
     {
         if (null === $value) {
+            if ($platform->getName() == 'oracle') {
+                return '';
+            }
+
```

Comment:
@cjbj done
https://bugs.php.net/bug.php?id=72524

Hope that is clear and simple enough.